awx/ui_next/src/screens/Credential/shared/CredentialForm.jsx

```diff
@@ -84,6 +84,16 @@ function CredentialFormFields({
     [setFieldValue]
   );
 
+  function verifyGalaxyCredential(credentialId) {
```

Comment:
I'd suggest changing this to `isGalaxyCredential` or `isGalaxyCredentialType` for readability.